spec/cli_spec.rb

```diff
@@ -1,9 +1,62 @@
 RSpec.describe TurboTests::CLI do
-  subject(:output) { `bundle exec turbo_tests -f d #{fixture}`.strip }
+  let(:bin) { Pathname(__dir__).join("../bin/turbo_tests").expand_path }
```

Comment:
This isn't necessary, but nice to bypass bundler overhead if we can.

Reply:
Is `bin` used?